tests/test_server.py

```diff
@@ -864,9 +864,33 @@ async def test_get_resource_actions(postgresql_client, client, clienthelper, ser
     )
     assert result.code == 200
 
+    #  adding a resource action with its change field set to "created" to test the get_resource_actions
+    #  filtering on resources with changes
+    rvid_r1_v1 = "std::File[agent1,path=/etc/file1],v=1"
+    await data.Resource.new(
+        environment=uuid.UUID(environment),
+        status=const.ResourceState.available,
+        resource_version_id=rvid_r1_v1,
+        attributes={"purge_on_delete": False, "purged": True, "requires": []},
+    ).insert()
```

Comment:
Why not add this resource using the `put_version()` endpoint like all the other resources? Doing this breaks the fundamental invariant of the code base that no resources can be added to an existing configuration model. This might cause all sorts of instabilities on the test.